picard/releasegroup.py

```diff
@@ -86,8 +90,8 @@ def _parse_versions(self, document):
             release = {
                 "id":      node['id'],
                 "year":    node['date'][:4] if "date" in node else "????",
-                "country": "+".join(countries) if countries
-                           else node.get('country', '') or "??",
+                "country": limited_join(countries, 10, '+', '…') if countries
```

Comment:
Is 10 fine, or even lower this? Also I like the `limited_join` function and how it drops in the middle, but is it actually useful for this case? Does it help to see the last entries or would rather only showing the first 10 + "..." be better?

Reply:
Actually, I was thinking about adding another parameter, to place the "..." at the end instead.
But since we have no idea what will be or not in the list, it doesn't change much